### PR TITLE
Add crowdsourced locations

### PR DESCRIPTION
**Pull request by wseagar**
Data is just hardcoded into frontend for now

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -45,14 +46,13 @@ function App() {
...
           />
-
-          <WalkInSection coords={coords} radiusKm={radiusKm} />
-
           <BookingSection
             coords={coords}
             radiusKm={radiusKm}
             setLastUpdateTime={setLastUpdateTime}
           />
+          <WalkInSection coords={coords} radiusKm={radiusKm} />
```
Comment (oeed) [inline src/App.tsx:54]: Do we want these underneath the calendar? As the calendar is much larger than walk-in and crowd sourced locations I can foresee that many people wouldn't scroll far enough to know that these sections are there.

File: src/crowdsourced/modal/CancelNotice.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent } from "react";
+
+export const CancelBookingNotice: FunctionComponent<{ className: string }> = ({
```
Comment (oeed) [inline src/crowdsourced/modal/CancelNotice.tsx:3]: We can probably use the existing version of this component, don't need to double up. Maybe move the original one to another folder?

File: src/crowdsourced/CrowdsourcedSection.tsx [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      <CrowdsourcedModal
+        clearSelectedLocation={clearSelectedLocation}
+        location={
+          "ok" in locations && selectedLocationIndex !== undefined
+            ? locations.ok[selectedLocationIndex]
+            : undefined
+        }
+      />
+      <h2 className="WalkSection">
+        Crowdsourced Vaccination Locations<strong> - Open Today</strong>
```
Comment (oeed) [inline src/crowdsourced/CrowdsourcedSection.tsx:44]: Do we want 'crowdsourced' as the title? Maybe it's good to make it clear, but to plays devils advocate would it make more sense to the user that it's simply a location that doesn't support the bookmyvaccine booking system? 

To the user, it doesn't matter how we got the data, what matters is how they book and find slots.

**Review by oeed — CHANGES_REQUESTED** [review]

**Comment by oeed** [comment]
@waltzaround your thoughts on these notes?

**Comment by wseagar** [comment]
Going to hold off on this until we get i18n in

File: src/crowdsourced/CrowdsourcedSection.tsx (hunk 3 above)
Comment (wseagar) [reply src/crowdsourced/CrowdsourcedSection.tsx:45]: I agree but I haven't heard a better name yet

"Other Vaccination Centres" ?

I think we can keep the code calling it Crowdsourced but I agree we need to change that wording before we ship

**Review by oeed — APPROVED** [review]

**Pull request merged**